src/indent.c

```diff
@@ -779,54 +779,6 @@ string_display_width (Lisp_Object string, Lisp_Object beg, Lisp_Object end)
 #endif /* 0 */
 
 
-DEFUN ("indent-to", Findent_to, Sindent_to, 1, 2, "NIndent to column: ",
-       doc: /* Indent from point with tabs and spaces until COLUMN is reached.
-Optional second argument MINIMUM says always do at least MINIMUM spaces
-even if that goes past COLUMN; by default, MINIMUM is zero.
-
-The return value is the column where the insertion ends.  */)
-  (Lisp_Object column, Lisp_Object minimum)
-{
-  EMACS_INT mincol;
-  register ptrdiff_t fromcol;
-  int tab_width = SANE_TAB_WIDTH (current_buffer);
-
-  CHECK_NUMBER (column);
```

Comment:
`CHECK_NUMBER` validates that column is an integer. But it does not check for positive or negative. So `EmacsInt` is correct. If instead either `NATNUMP` or `CHECK_NATNUM` is used then `EmacsUint` is appropriate.